packages/core/resources/icons/icons.json

```diff
@@ -2840,5 +2840,26 @@
     "tags": "sidebar, layout, spreadsheet",
     "group": "table",
     "content": "\\e778"
+  },
+  {
+    "name": "Tick circle",
+    "className": "pt-icon-tick-circle",
+    "tags": "tick mark, sign, ok, approved, success",
+    "group": "action",
+    "content": "\\e779"
+  },
+  {
+    "name": "Prescription",
```

Comment:
this doesn't feel like a useful UI icon, it's specific to certain use cases. why do we need?